api/src/main/scala/org/virtuslab/ideprobe/ProbeExtensions.scala

```diff
@@ -173,3 +170,37 @@ trait ProbeExtensions {
     }
   }
 }
+
+object ProbeExtensions {
+  private class DeletingVisitor(root: Path) extends SimpleFileVisitor[Path] {
+    override def visitFile(file: Path, attrs: BasicFileAttributes): FileVisitResult = {
+      if(!Files.isDirectory(file))  Files.delete(file)
```

Comment:
You should be able to check if this is directory by calling a method on attrs. That would save us an IO call.

Reply:
Done